src/app/shared/constants/dishes.constants.ts

```diff
@@ -131,7 +131,7 @@ export const DISHES: Dish[] = [
     {
         id: '3',
         name: 'ElaiCheese Cake',
-        image: '/conFusion/assets/images/elaicheesecake.png',
+        image: '/images/elaicheesecake.png',

```

Comment:
looks like /images is duplicated and should be part of the constants

Reply:
Will be done in next PR